sql/5_0_2-to-5_0_3_upgrade.sql

```diff
@@ -419,4 +419,15 @@ CREATE TABLE `pro_assessments` (
 
 #IfNotRow2D list_options list_id LBF_Validations option_id future_date
 INSERT INTO `list_options` (`list_id`,`option_id`,`title`,`notes`, `seq`) VALUES ('LBF_Validations','future_date','Future Date','{\"futureDate\":{\"message\":\"must be future date\"}}','32');
+
+#IfNotRow2D list_options list_id lists option_id Sort_Direction
+INSERT INTO `list_options` (`list_id`, `option_id`, `title`, `seq`, `is_default`, `activity`) VALUES ('lists', 'Sort_Direction', 'Sort Direction', 306, 1, 1);
```

Comment:
would make seq 1 here and is_default 0 (since that is what is done on most of the list titles)

Reply:
Fixed